activesupport/test/core_ext/string_ext_test.rb

```diff
@@ -573,6 +573,10 @@ def test_string_to_time
     end
   end
 
+  def test_timestamp_string_to_time
+    assert_equal "argument out of range", "1604326192".to_time
+  end
```

Comment:
Your unit test isn't testing the error is raised so the active support build is failing. It needs to be changed to something like this:

```suggestion
  def test_timestamp_string_to_time
    error = assert_raise ArgumentError do
      "1604326192".to_time
     end
     
    assert_equal "argument out of range", error.message
  end
```

Reply:
Thanks for the suggestion. I have updated accordingly.

Reply:
@eileencodes I implemented the changes, but the build still fails. From the error message, it seems not related with my changes. How to fix it?

Reply:
Can you pull master and then rebase? The sqlite3_mem failure looks like a failure I fixed earlier yesterday and the active job ones don't look related to your changes.

Reply:
It works!!!